drivers/include/net/netdev2_eth.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @ingroup     drivers_netdev_netdev2
+ * @{
+ *
+ * @file
+ * @brief       Definitions for netdev2 common ethernet code
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef NETDEV2_ETH_H
+#define NETDEV2_ETH_H
+
+#ifdef __cplusplus
```

Comment:
`#include "net/netdev2.h` missing here (just got a warning while using this for the `enc28j60` driver...)